cmd/runtimetest/main.go

```diff
@@ -271,6 +271,62 @@ func validateDefaultFS(spec *rspec.Spec) error {
 	return nil
 }
 
+func validateLinuxDevices(spec *rspec.Spec) error {
+	logrus.Debugf("validating linux devices")
+
+	for _, device := range spec.Linux.Devices {
+		fi, err := os.Stat(device.Path)
+		if err != nil {
+			return err
+		}
+		fStat := fi.Sys().(*syscall.Stat_t)
```

Comment:
This `Stat` with its type/major/minor checks (and probably the `Stat` existence check too) should get pulled out into it's own helper (`func validateDevice(path string, device *rspec.Device) error`?) so we can use it in `validateDefaultDevices` as well.